README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
+1. Press the space bar to begin.
+2. Use the up and down arrow keys to rotate the shape.
```

Comment:
It might be helpful to include whether the shape will be rotated clockwise or counter-clockwise if up is pressed. That way the user knows which direction to expect based on their input.